packages/build-tools/src/ios/pod.ts

```diff
@@ -1,12 +1,40 @@
 import path from 'path';
 
+import fs from 'fs-extra';
 import { Ios } from '@expo/eas-build-job';
 import spawn from '@expo/turtle-spawn';
 
 import { BuildContext } from '../context';
 
 export async function installPods<TJob extends Ios.Job>(ctx: BuildContext<TJob>): Promise<void> {
   const iosDir = path.join(ctx.reactNativeProjectDirectory, 'ios');
+
+  if (ctx.env.EAS_BUILD_COCOAPODS_CACHE_URL) {
+    ctx.logger.info('Running using experimental CocoaPods cache proxy...');
+    const podFilePath = path.join(iosDir, 'Podfile');
+    const originalPodFileContentBuffer = await fs.readFile(podFilePath);
+    const originalPodFileContent = originalPodFileContentBuffer.toString('utf-8');
+
+    if (
+      originalPodFileContent.search(/source( )+('|")https:\/\/cdn.cocoapods.org(\/)*('|")/) !== -1
+    ) {
+      ctx.logger.info('Replacing CocoaPods CDN source with proxy...');
+      await fs.writeFile(
+        podFilePath,
+        originalPodFileContent.replace(
+          /source( )+('|")https:\/\/cdn.cocoapods.org(\/)*('|")/,
+          `source "${ctx.env.EAS_BUILD_COCOAPODS_CACHE_URL}"`
+        )
+      );
+    } else {
+      ctx.logger.info('Adding proxy as a source...');
```

Comment:
This statement will not mean anything to users, you need to make it more precise or sth more generic.
e.g.
Adding EAS CocoaPods cache to the Podfile / 'Replacing CocoaPods CDN with EAS CocoaPods cache in the Podifle

or sth more generic

Enable EAS CocoaPods cache.

Also `...` should be used only if sth might take a bit, updating that file will be very quick

Reply:
I think I will drop all these logs